citrination_client/client.py

```diff
@@ -48,12 +51,12 @@ def __init__(self, api_key=None, site=None, suppress_warnings=False, proxies=Non
         self.data = DataClient(api_key, site, suppress_warnings=suppress_warnings, proxies=proxies)
         self.data_views = DataViewsClient(api_key, site, suppress_warnings=suppress_warnings, proxies=proxies)
 
-        clients = [self.models, self.search, self.data, self.data_views]
+        # clients = [self.models, self.search, self.data, self.data_views]
```

Comment:
Why's this commented out?

Reply:
I ran into a conflict with the little lambda injector below.